.github/workflows/npmpublish.yml

```diff
@@ -35,16 +35,10 @@ jobs:
           node-version: 12
           registry-url: https://registry.npmjs.org/
 
-      - name: Install
-        run: npm ci
-
-      - name: Build
-        run: npm run build:prod
-
       - name: Install Production
         run: npm ci --production
```

Comment:
I think that will no work, you need the dev dependencies to build. That's why first `npm ci` and then build and then `npm ci --production`